src/resource/user/user.service.ts

```diff
@@ -52,6 +58,32 @@ export class UserService {
     });
   }
 
+  async verifyAccountSendEmail({ id }: VerifyAccountSendEmailInput) {
+    const user = await this.findById(id);
+    const { email, username } = user;
+    const verifyEmailLink = `${process.env.CLIENT_BASE_URL}verify-account/${id}`;
+    const subject = 'Please verify your A-Comosus account';
+    const emailContent = `<b>Hi ${username} 👋</b> 
+    <p>Please verify your A-Comosus account following the link: </p> 
+    <a>${verifyEmailLink}</a>
+    <br>
+    <br>
+    <b>A-COMOSUS🍍</b>`;
+
+    const result = await this.axiosService.sendEmail({
+      email,
+      subject,
+      emailContent,
+    });
+
+    this.logger.log(
+      'axiosService.sendEmail of verify user email result ...',
+      result,
+    );
+    if (!result) return false;
```

Comment:
I think it's better to log a separate `logger.error` for this?

Reply:
updated with log err